source/includes/fundamentals/code-snippets/Filters.java

```diff
@@ -153,6 +168,7 @@ private void setupPaintCollection() {
         String [] p7a = {"B", "C"};
         String [] p8a = {"A", "D"};
 
+        // Creates a series of documents and inserts them into the "paint" collection
```

Comment:
Suggestion:
I think this should be omitted since it's setup code meant only for tech writers